GAMEPLAY.md

```diff
@@ -8,6 +8,9 @@ You can see the current scoreboard by typing `,score`. It is also shown after th
 
 See the latest updates to KMQ with `,news`. 
 
+## Game Modes
+See who can survive the longest into a KMQ game with elimination mode. Using `,play elimination x`, everyone starts with `x` lives and the last one alive wins. The first one to guess right keeps a life. Use elimination mode in conjunction with `,timer` to raise the pressure! Elimination mode and `,goal` are incompatible with each other.
```

Comment:
`The first one to guess right keeps a life` => `Guessing correctly will save your life, while everyone else loses one.`
`Elimination mode and ',goal' are incompatible with each other.` => ',goal' cannot be used in elimination mode